style-guide/secondary.html

```diff
@@ -162,7 +162,7 @@
 							<h1>Strategic Communication</h1>
 						</header>
 						<div class="column one" style="">
-							<h4>Department of Strategic Communication</h4>
+							<p class="sub-callout">Department of Strategic Communication</p>
```

Comment:
Is this an editorial instruction in production or do we need to account for this with custom code?

Reply:
Right now it is instruction. Are you asking if it should be a class available to added to a paragraph within TinyMCE?

Reply:
Yeah. If that makes sense to help with content entry, it could be something to try.

Reply:
We should ask @allenjohns on Monday if there is any concern with this being something easily available for all content editors to use.